src/helpers/utils.js

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable valid-jsdoc */
 import bcrypt from 'bcrypt';
 import jwt from 'jsonwebtoken';
+import slug from 'slug';
```

Comment:
Please remove `slug`

Reply:
Thank you for this review. This has been resolved.